rest/src/main/java/eu/openg/guessnumberapi/rest/entity/converter/GuessResponseConverter.java

```diff
@@ -1,15 +1,26 @@
 package eu.openg.guessnumberapi.rest.entity.converter;
 
-import eu.openg.guessnumberapi.rest.entity.RestGuessResponseEntity;
+import eu.openg.guessnumberapi.rest.entity.RestGuessResponse;
 import eu.openg.guessnumberapi.usecase.api.BoundaryGuessResponse;
+import eu.openg.guessnumberapi.usecase.api.BoundaryGuessResultStatus;
 
-import static java.util.Objects.nonNull;
+import java.util.Optional;
+
+import static java.util.Objects.isNull;
 
 public class GuessResponseConverter {
 
-    public RestGuessResponseEntity convert(BoundaryGuessResponse guessResponse) {
-        if (nonNull(guessResponse))
-            return new RestGuessResponseEntity(guessResponse.getMessage(), guessResponse.getNumberOfGuesses());
-        return null;
+    public RestGuessResponse convert(BoundaryGuessResponse guessResponse) {
+        if (isNull(guessResponse))
+            return null;
+        BoundaryGuessResultStatus status = guessResponse.getStatus();
+        if (status == BoundaryGuessResultStatus.CORRECT)
+            return new RestGuessResponse(null, guessResponse.getNumberOfGuesses());
```

Comment:
gali pasidaryti tuščią interface RestGuessResponse, kuris turėtų dvi implementacijas: RestGameFinishedResponse ir RestGameOngoingResponse (namingą gali keisti). vienas turėtų number of guesses, o kitas string status